src/stellarmesh/moab.py

```diff
@@ -210,6 +219,38 @@ def surf_sense(self, volumes: list[Optional[DAGMCVolume]]):
             if vol is not None:
                 self.model._core.add_parent_child(vol.handle, self.handle)
 
+    @property
+    def boundary(self) -> Optional[str]:
+        """Name of the boundary condition assigned to this surface."""
+        for group in self.groups:
+            if self in group and group.name.startswith("boundary:"):
```

Comment:
The first condition is superfluous since `self.groups` only gives groups that `self` is in
```suggestion
            if group.name.startswith("boundary:"):
```